tasks/README.md

```diff
@@ -98,15 +101,23 @@ Any parameter that does not specify a type will accept any value (default type i
 
 If a parameter type accepts `null` the task runner will accept either a `null` value or the absence of the property. Task authors must accept missing properties for nullable parameters. Task authors must not differentiate between absent properties and properties with `null` values.
 
+### Metaparameters
+
+In addition to the tasks parameters, the task runner may inject metaparameters prefixed by '_'. These include `_noop` and `_task`.
```

Comment:
Is `_task` just the name of the current task?

Reply:
correct. You can have the same file implement multiple tasks and behave slightly differently.

for example
package::install
package::status
package::uninstall
could all use the package script